app/players/elves/elf.py

```diff
@@ -0,0 +1,15 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
The `Elf` class should be abstract because it does not implement all abstract methods from the `Player` class. Inherit it from the `ABC` as well.

Reply:
the same